src/requests/authorizertatxrequest.cpp

```diff
@@ -181,6 +182,20 @@ bool validateAuthResponse(const AuthorizeRtaTxResponse &arg, const SupernodePtr
     return r1 && r2;
 }
 
+Status storeRequestAndReplyOk(const Router::vars_t& vars, const graft::Input& input,
```

Comment:
Are you need it? You have template method for this.

Reply:
Here it doesn't fit the protocol - template method is for building responses to the client, but this one is to build response to cryptonode (result.Status vs result.Result). Probably we'll fix protocol later